building_sim_plugins/building_ignition_plugins/src/crowd_simulator.cpp

```diff
@@ -0,0 +1,330 @@
+#include <regex>
+#include <cstdlib>
+
+#include <sdf/Actor.hh>
+
+#include <ignition/math/Pose3.hh>
+
+#include <ignition/gazebo/components/Name.hh>
+#include <ignition/gazebo/components/Model.hh>
+#include <ignition/gazebo/components/Actor.hh>
+#include <ignition/gazebo/components/Pose.hh>
+#include <ignition/gazebo/components/Static.hh>
+
+#include "crowd_simulator.hpp"
```

Comment:
Similar to above, we can consider shifting the header into `src` since we are not exposing any public API